### PR TITLE
Typography

### PR DESCRIPTION
**Pull request by arundhatij12**

### CODE REVIEW

File: input.css [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+
+.list {
+  display: flex;
+  flex-direction: column;
+  margin-top: 20px;
+}
+
+.item {
+  display: flex;
+  padding: 13px 36px;
```
Comment (srejitk) [inline input.css:58]: Would suggest using Relative units instead of px

File: input.css [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+}
+
+.content2 {
+  display: flex;
+  gap: 10px;
+  padding: 8px;
+  width: 800px;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
```
Comment (srejitk) [inline input.css:80]: The flex properties can be a utility class

File: input.css [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+
+#desc3 {
+  color: rgb(184, 32, 32);
+  margin-top: 20px;
+}
+
+.input {
+  background-color: rgb(77, 177, 193);
+  height: 120px;
+  width: 300px;
```
Comment (srejitk) [inline input.css:119]: Save the colors as variables and give the height and width in relative units

**Review by srejitk — APPROVED** [review]
Suggesting small changes. Just use of utility classes to prevent rewriting the same code.

File: input.css [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+}
+
+header {
+  background-color: rgba(114, 195, 202, 0.575);
+  height: 80px;
+}
+
+a {
+  text-decoration: none;
+  color: #3d6dc5;
```
Comment (shuntzhere) [inline input.css:13]: It was suggested to use variables for colors.

File: input.html [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+          <input type="radio" name="rad" id="" />Select1
+          <input type="radio" name="rad" id="" />Select2
+        </div>
+        <img class="image-code" src="code/inputbox1.png" alt="code" />
+        <div class="desc">Checkboxes</div>
+        <div class="input">
+          <br />
+          <input type="checkbox" name="" id="" />box1
+          <input type="checkbox" name="" id="" />box2
+        </div>
```
Comment (shuntzhere) [inline input.html:86]: Don't leave the attributes empty.

File: input.html [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+      </div>
+    </section>
+
+    <div class="tab">
+      <a href="image.html"> <button>PREVIOUS</button></a>
+      <a href="typography.html"> <button>NEXT</button></a>
+    </div>
+
+    <footer class="foo">
+      <a href="" class="links">
```
Comment (shuntzhere) [inline input.html:97]: The link is missing here.

File: typography.html [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+    <script
+      src="https://kit.fontawesome.com/946eb6d076.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <header>
+      <div class="head">
+        <div>
+          <a class="title" href="">
```
Comment (shuntzhere) [inline typography.html:18]: Give a # incase there's no link to redirect.

**Review by shuntzhere — APPROVED** [review]
Suggested some changes to go with.

**Pull request merged**